tools/lightning-checkpoints.md

```diff
@@ -0,0 +1,95 @@
+---
+title: TiDB Lightning Checkpoints
+summary: Use checkpoints to avoid redoing the previously completed tasks before the crash.
+category: tools
+---
+
+# TiDB Lightning Checkpoints
```

Comment:
```suggestion
# TiDB-Lightning Checkpoints
```

According to https://github.com/pingcap/tidb-lightning/pull/80#discussion_r232165326 we should hyphenate it